web/src/components/pages/contribution/success.tsx

```diff
@@ -31,6 +31,26 @@ const GoalPercentage = ({
   </span>
 );
 
+const ContributeMoreButton = (props: {
+  hasAccount: boolean;
+  children: React.ReactNode;
+  onClick: () => any;
+}) =>
+  props.hasAccount ? (
```

Comment:
are we getting hasAccount as props? may be not !

Reply:
It was moved into the props here so we can declare `ContributeMoreButton` outside the scope of the parent function component.

This prevents us from redefining the `ContributeMoreButton` every time the parent function component renders.

Reply:
Ah, got it below. Nice